hw2/04-background.js

```diff
@@ -1 +1,33 @@
-// Add your code here
+// 04-background.js
+
+const body = document.body;
+const input = document.querySelector('input');
+const startButton = document.getElementById('button');
+
+function getRandomColor() {
+  const colors = [
+    '#FF5733',
+    '#33FF57',
+    '#5733FF',
+    '#FF33A6',
+    '#33A6FF',
+    '#FFA633',
+  ];
+
+  const randomIndex = Math.floor(Math.random() * colors.length);
+
+  return colors[randomIndex];
+}
+
+startButton.addEventListener('click', () => {
+  const intervalInSeconds = Number(input.value);
+
+  if (Number.isNaN(intervalInSeconds) || intervalInSeconds <= 0) {
```

Comment:
I think instead of adding a check here you can leverage the benefit of the type attribute of the input tag, it will only accept numbers.